### ISSUE TITLE
Address JSON schema deprecation warning

### ISSUE THREAD

**Issue by cortadocodes** [octue/octue-sdk-python#368]
```
DeprecationWarning: The metaschema specified by $schema was not found. Using the latest draft to validate, but this will raise an error in the future.
  cls = validator_for(schema)
```

**Comment by thclark** [comment]
Yeah we should be able to resolve this by updating the $schema reference to the correct draft. One of those things I never did. If we wanted to we could re-host the draft on octue's CDN for improved performance, but perhaps that's premature optimisation.

**Comment by cortadocodes** [comment]
Cool I'm guessing the source of this is the various schemas in `twined`?

**Comment by cortadocodes** [comment]
@thclark it turns out we've actually been using the `http://json-schema.org/draft-07/schema#` JSON metaschema because the schema URL was slightly incorrect, leading to the latest metaschema version according to the `jsonschema` package version being used. We've been using `jsonschema==3.2.0` for which the latest metaschema is `draft-07`. Should we update to the latest metaschema?
